ui/app/pages/swaps/main-quote-summary/main-quote-summary.js

```diff
@@ -1,94 +1,72 @@
-  isBestQuote,
   sourceValue,
   sourceSymbol,
   sourceDecimals,
+  sourceIconUrl,
   destinationValue,
   destinationSymbol,
   destinationDecimals,
+  destinationIconUrl,
 }) {
 
-  const t = useContext(I18nContext)
-
   const sourceAmount = toPrecisionWithoutTrailingZeros(calcTokenAmount(sourceValue, sourceDecimals).toString(10), 12)
   const destinationAmount = calcTokenAmount(destinationValue, destinationDecimals)
 
   const amountToDisplay = formatSwapsValueForDisplay(destinationAmount)
-  const fontSizeScore = getFontSizeScore(amountToDisplay, destinationSymbol)
-  const [numberFontSize, symbolFontSize] = getFontSizes(fontSizeScore)
-  const lineHeight = getLineHeight(fontSizeScore)
-
+  const amountDigitLength = amountToDisplay.match(/\d+/gu).join('').length
+  const [numberFontSize, lineHeight] = getFontSizesAndLineHeights(amountDigitLength)
   let ellipsedAmountToDisplay = amountToDisplay
-  if (fontSizeScore > 20) {
-    ellipsedAmountToDisplay = `${amountToDisplay.slice(0, amountToDisplay.length - (fontSizeScore - 20))}...`
+  if (amountDigitLength > 20) {
+    ellipsedAmountToDisplay = `${amountToDisplay.slice(0, 20)}...`
   }
 
   return (
     <div className="main-quote-summary">
-      <div
-        className={classnames('main-quote-summary__quote-backdrop', {
-          'main-quote-summary__quote-backdrop-with-top-tab': isBestQuote,
-        })}
-      >
-        <QuoteBackdrop withTopTab={isBestQuote} />
-      </div>
-      <div className="main-quote-summary__best-quote">
-        {isBestQuote && <SunCheckIcon />}
-        <span>{isBestQuote && t('swapsBestQuote')}</span>
-      </div>
       <div className="main-quote-summary__details">
         <div className="main-quote-summary__quote-details-top">
-          <span className="main-quote-summary__quote-small-white">
-            {t('swapsConvertToAbout', [<span className="main-quote-summary__bold" key="main-quote-summary-bold-1">{`${sourceAmount} ${sourceSymbol}`}</span>])}
-          </span>
+          <div className="main-quote-summary__source-row">
+            <span title={formatSwapsValueForDisplay(sourceAmount)}>{ formatSwapsValueForDisplay(sourceAmount) }</span>
+            <UrlIcon
+              url={sourceIconUrl}
+              className="main-quote-summary__icon"
+              name={sourceSymbol}
+              fallbackClassName="main-quote-summary__icon-fallback"
+            />
+            <span title={sourceSymbol}>{ sourceSymbol }</span>
+          </div>
+          <img
+            className="main-quote-summary__down-arrow"
+            src="images/down-arrow-grey.svg"
+          />
+          <div className="main-quote-summary__destination-row">
+            <UrlIcon
+              url={destinationIconUrl}
+              className="main-quote-summary__icon"
+              name={destinationSymbol}
+              fallbackClassName="main-quote-summary__icon-fallback"
+            />
+            <span>{ destinationSymbol }</span>
```

Comment:
can we add a className to these spans so that the scss can target them by a classname and not by an element type? Hopefully, that would lead to more easily understanding of the CSS when coming back to it/new dev onboarding.

Reply:
Done in e3841a2e6